### DISCUSSION TITLE
Bulk Downloader for Reddit v2.4.2

### DISCUSSION THREAD

**Discussion by Serene-Arc** [Serene-Arc/bulk-downloader-for-reddit#538] [General]
Included in this release:

- Bug fixes for Erome module
- Moved to yt-dpl so youtube support and video support enhanced
- Other bug fixes

**Full Changelog**: https://github.com/aliparlakci/bulk-downloader-for-reddit/compare/v2.4.1...v2.4.2

<hr /><em>This discussion was created from the release <a href='https://github.com/aliparlakci/bulk-downloader-for-reddit/releases/tag/v2.4.2'>Bulk Downloader for Reddit v2.4.2</a>.</em>

**Comment by ghost** [comment]
BDFR is downloading videos incorrectly for me and I suspect it might have something to do with the switch to yt-dpl. All video content is missing the period before the file type. For example, MyVactionVideo.mp4 when downloaded becomes MyVactionVideomp4. I can manually add the period back to videos, but that's too much hassle for 1000's of videos. Let me know what info I can give you to help identify this issue. 

I do have ffmpeg installed correctly on Win 10 (cmd ffmpeg brings up info by default).

**Reply by Serene-Arc** [reply]
If you could open up a bug report, I'll start working on finding the solution.

**Reply by Serene-Arc** [reply]
If you could find a submission ID and the corresponding URL that downloads incorrectly, that would be a big help

**Comment by ghost** [comment]
Had the same issue. Worse on some accounts, less on others. User 1O9 (NSFW) has the issue with all videos. Sub ID _gphmnr & _d0oir2 for SFW specific examples. User shiritrap (NSFW) works 95% of their videos. Only 4 videos have the issue.

This is the code I use:
python3 -m bdfr download C:\Users\0rpheus\Downloads\Test --submitted --no-dupes --folder-scheme {REDDITOR} --file-scheme {REDDITOR}_{SUBREDDIT}_{TITLE}_{POSTID} -u shiritrap -u 1O9 u- hereonneptune

I suspect it one of the video hosting providers?

Also, user hereonneptune (NSFW) causes a crash everytime reguardless of if it is just that single account or part of a group.

**Reply by Serene-Arc** [reply]
Thanks, I'll look into it
